### DISCUSSION TITLE
No media to process... / Arhiiv ERR (Archive ERR)

### DISCUSSION THREAD

**Discussion by madrulane** [aclap-dev/video-downloadhelper#56] [Help and Q&A] [answered]
### Your question

As the title says... [link](https://arhiiv.err.ee/video/vaata/inimene-ja-kompuuter)

![24-02-09 15_48_35-Inimene ja kompuuter _ Videoarhiiv _ ERR and 62 more pages - Personal - Microsof](https://github.com/aclap-dev/video-downloadhelper/assets/21111550/4a8f0247-135b-4fc6-b0a7-40f34cf4629d)
(video was playing during the screenshot, of course)

### Addon & CoApp version and details

win10, everything latest

**Comment by paulrouget** [comment, marked as answer]
This is now working in 8.2 ([version beta](https://github.com/aclap-dev/video-downloadhelper/wiki/Beta-versions)).

**Comment by madrulane** [comment]
And indeed it is, thanks!
